src/ui/components/filters/sortoptionscomponent.js

```diff
@@ -43,6 +43,33 @@ export default class SortOptionsComponent extends Component {
         }
       }
     });
+
+    this.core.storage.registerListener({
+      eventType: 'update',
+      storageKey: StorageKeys.HISTORY_POP_STATE,
+      callback: () => {
+        const persistedOptionIndex = this.getPersistedOptionIndex();
+        this._updateSelectedOption(persistedOptionIndex);
+        this.setState();
+      }
+    });
+  }
+
+  /**
+   * Returns the option index matching the persisted sortBys, if one exists.
+   *
+   * @returns {number|undefined}
+   */
+  getPersistedOptionIndex () {
+    const persistedSortBys = this.core.storage.get(StorageKeys.SORT_BYS) || [];
```

Comment:
Why is persistedSortBys an array? Aren't we just persisting the selected one?

Reply:
The api parameter takes in an array, it's just that the SDK is only set up to let you sort by one "sort by" at a time

Reply:
The SDK also internally stores an array for SORT_BYS, though it'll only ever have one element, since SortOptions only ever calls core.setSortBys() with one element. (I guess you could directly call core.setSortBys with multiple elements in your config/onReady)